test/cli/bootstrap.js

```diff
@@ -8,7 +8,8 @@ const runOnAndOff = require('../utils/on-and-off')
 describe('bootstrap', () => runOnAndOff((thing) => {
   let ipfs
 
-  before(() => {
+  before(function () {
+    // this.timeout(30 * 1000)
```

Comment:
If there is no need for the timeout, revert back to an arrow function and remove the comment

Reply:
@diasdavid yeah, I'm getting some random test fails and trying to isolate as to whats causing them, this commented timeouts are a WIP, disregard for now please.